server.js

```diff
@@ -4,16 +4,26 @@
 // @noflow
 const webpack = require('webpack');
 const WebpackDevServer = require('webpack-dev-server');
+const express = require('express');
 const config = require('./webpack.config');
 const { oneLine, stripIndent } = require('common-tags');
 const port = process.env.FX_PROFILER_PORT || 4242;
 const host = process.env.FX_PROFILER_HOST || 'localhost';
 const fs = require('fs');
 const path = require('path');
+const yargs = require('yargs');
+const { hideBin } = require('yargs/helpers');
 const localConfigExists = fs.existsSync(
   path.join(__dirname, './webpack.local-config.js')
 );
 
+const argv = yargs(hideBin(process.argv))
+  .command('* [<profile>]', 'Open Firefox Profiler, on <profile> if included.')
```

Comment:
On yargs 17.6.2, omitting the `*` doesn't work -- without it, yargs refuses to process a `profile` arg that is given on the command line. I did think about using `open` instead of the default command, but I don't see any other function for `server.js` other than to (eventually) open profiles.

Sorry about the `<>`, initially the `profile` arg was required, and I just slapped on the `[]` as it seemed the natural thing to do, without looking at the documentation. Thanks for the catch!